src/tpt_reports/tpt_reports.py

```diff
@@ -45,8 +52,85 @@ def load_json_file(file_path):
         return None
 
 
-# Issue #4 - Add ReportLab code and library
-# TODO: Add in the ReportLab code, library, parsing logic and remove this comment.
+# Issue #21 - test parse_json()
+# TODO: Add unit tests for following logic and remove this comment.
+def parse_json(data):
+    """Parse JSON object for values to report."""
+    payloads_list = []
+    payloads_meta = {}
+    num_payloads = 0
+    host_blocked = 0
+    host_not_blocked = 0
+    border_blocked = 0
+    border_not_blocked = 0
```

Comment:
Always prefer to sort things like this alphabetically for easier maintenance:
```suggestion
    border_blocked = 0
    border_not_blocked = 0
    host_blocked = 0
    host_not_blocked = 0
    num_payloads = 0
    payloads_list = []
    payloads_meta = {}
```